apps/calendar/components/calendar/week/WeekColumnsList.tsx

```diff
@@ -126,7 +129,7 @@ const WeekColumnsList = () => {
         );
       return { ...lead, difference: [...differenceDay] };
     });
-
+//Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.
```

Comment:
//Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.